### PR TITLE
Update `p2p-private-key-file` command line option description

### PR DESCRIPTION
**Pull request by courtneyeh**

### CODE REVIEW

File: docs/reference/cli/index.md [hunk 1]
```diff
@@ -2139,6 +2139,14 @@ p2p-private-key-file: "/home/me/me_node/key"
 
 File containing the [node's private key](../../concepts/p2p-private-key.md).
 
+If there is not a file present at the path provided, one will be created with a new p2p private key generated and stored inside.
+
+:::important
+
+Ensure you provide a file path and not a directory as no file will be read or created.
```
Comment (joaniefromtheblock) [inline docs/reference/cli/index.md:2146]:
```suggestion
Make sure you specify a file path and not a directory. Teku doesn't read or create files in directories.
```

File: docs/reference/cli/index.md [hunk 2]
```diff
@@ -2139,6 +2139,14 @@ p2p-private-key-file: "/home/me/me_node/key"
 
 File containing the [node's private key](../../concepts/p2p-private-key.md).
 
+If there is not a file present at the path provided, one will be created with a new p2p private key generated and stored inside.
```
Comment (joaniefromtheblock) [inline docs/reference/cli/index.md:2142]:
```suggestion
If a file doesn't exist at the specified path, Teku creates a new file and p2p private key to store inside.
```

File: docs/reference/cli/index.md (hunk 1 above)
Comment (courtneyeh) [reply docs/reference/cli/index.md:2146]: I mean it does create files in directories, but you need to specify the path with the file name, not just the directory it is in.

Comment (joaniefromtheblock) [reply docs/reference/cli/index.md:2146]: oh! Thank you for the clarification @courtneyeh

File: docs/reference/cli/index.md [hunk 3]
```diff
@@ -1652,7 +1652,6 @@ Possible values are:
 | `gnosis` | Consensus layer | Production | Network for the [Gnosis chain](https://www.gnosis.io/) |
 | `holesky` | Consensus layer | Test | Multi-client testnet |
 | `sepolia` | Consensus layer | Test | Multi-client testnet |
-| `chiado` | Consensus layer | Test | Gnosis [testnet](https://docs.gnosischain.com/about/networks/chiado/) |
```
Comment (joaniefromtheblock) [inline docs/reference/cli/index.md:1655]: no link for this

**Review by joaniefromtheblock — APPROVED** [review]
lgtm

**Pull request merged**